libpod/options.go

```diff
@@ -268,6 +268,18 @@ func WithShmSize(size int64) CtrCreateOption {
 	}
 }
 
+// WithPrivileged sets the privileged flag in the container runtime
+func WithPrivileged(privileged bool) CtrCreateOption {
+	return func(ctr *Container) error {
+		if ctr.valid {
```

Comment:
This just looks odd, we've a valid container, so we raise an error.  Could we change this field name to finalized vs valid?  I don't know if that works or not, just thinking aloud.  Not a needed adjustment for this PR.